Remove this patch once Mesa is updated to the latest release

Not suitable for upstream

--- a/glx/glxdricommon.c	2019-02-12 12:52:53.219321827 -0800
+++ b/glx/glxdricommon.c	2019-02-12 12:54:58.061648159 -0800
@@ -259,6 +259,7 @@
 
 static const char dri_driver_path[] = DRI_DRIVER_PATH;
 
+#if 0
 /* Temporary define to allow building without a dri_interface.h from
  * updated Mesa.  Some day when we don't care about Mesa that old any
  * more this can be removed.
@@ -266,6 +267,7 @@
 #ifndef __DRI_DRIVER_GET_EXTENSIONS
 #define __DRI_DRIVER_GET_EXTENSIONS "__driDriverGetExtensions"
 #endif
+#endif
 
 void *
 glxProbeDriver(const char *driverName,
@@ -318,6 +320,7 @@
         goto cleanup_failure;
     }
 
+#if 0
     if (asprintf(&get_extensions_name, "%s_%s",
                  __DRI_DRIVER_GET_EXTENSIONS, driverName) != -1) {
         const __DRIextension **(*get_extensions)(void);
@@ -330,6 +333,8 @@
 
     if (!extensions)
         extensions = dlsym(driver, __DRI_DRIVER_EXTENSIONS);
+#endif
+    extensions = dlsym(driver, __DRI_DRIVER_EXTENSIONS);
     if (extensions == NULL) {
         LogMessage(X_ERROR, "AIGLX error: %s exports no extensions (%s)\n",
                    driverName, dlerror());
